<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelWidget</class>
 <widget class="QWidget" name="LabelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(85, 170, 255)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QWidget#scrollAreaWidgetContents{
background-color:#ECAD9E  ;
color:white;
border:none;
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>802</width>
          <height>534</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="labelName">
               <property name="font">
                <font>
                 <family>Mistral</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>标签名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="Detail">
               <property name="font">
                <font>
                 <family>MingLiU_HKSCS-ExtB</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>详情信息</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="Type">
               <property name="font">
                <font>
                 <family>Mistral</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>类型</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="AddTask">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>添加任务</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Monotype Corsiva</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>对应任务星级数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="StarSpinBox">
               <property name="suffix">
                <string>颗</string>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="Save">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Query">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>？</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
